plugins/scaffolder/src/components/fields/RepoUrlPicker/validation.ts

```diff
@@ -54,7 +54,14 @@ export const repoPickerValidation = (
           );
         }
       }
-      // For anything other than bitbucket and gerrit
+      if (integrationApi?.byHost(host)?.type === 'azure') {
```

Comment:
This should be an else-if

I could have added the condition to the bitbucket check, but I prefer this